keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -334,6 +451,64 @@ Disk usage will overall increase by the used memory of the container and the cha
 Checkpoint archive written to disk can optionally be compressed. The current implementation
 does not compress the checkpoint archive on disk.
 
+To avoid running out of disk space an operator has been introduced: <https://github.com/checkpoint-restore/checkpoint-restore-operator>
```

Comment:
is this component required to use the API?  Recommended?  Can the kubelet not check for sufficient space itself?

Reply:
> is this component required to use the API?

No

> Recommended?

I don't know.

>  Can the kubelet not check for sufficient space itself?

That was my first implementation. I was told this belongs in an operator.

Reply:
The idea is to separate the management of the checkpoints by an external entity to avoid adding a lot of GC tunables to the kubelet for this right now. This could be revisited in the future if/when this expands to kubernetes API.

Reply:
Can we explain it like so in this document, rather than pointing to some operator? By that I mean saying something along the lines of: the cluster administrator is responsible for monitoring disk usage and removing excess data.

Reply:
> The idea is to separate the management of the checkpoints by an external entity to avoid adding a lot of GC tunables to the kubelet for this right now. This could be revisited in the future if/when this expands to kubernetes API.

Which spot on disk will the cluster administrator need to watch?

Also, I think it's important to point out that I think this API *is* exposed via the kubernetes API via the clever use of a node/proxy endpoint.

Reply:
> Which spot on disk will the cluster administrator need to watch?

Added the information.

> Also, I think it's important to point out that I think this API is exposed via the kubernetes API via the clever use of a node/proxy endpoint.

Anything special I need to do to handle this? Or just pointing out that it is possible?